tests/functional/Sheet/Safe/OpenDocumentLoaderTest.php

```diff
@@ -82,4 +84,11 @@ public function testLoad()
             ['jean', 'dupont'],
         );
     }
+
+    public function pipelineRunner(): PipelineRunnerInterface
+    {
+        return new \Kiboko\Component\Pipeline\PipelineRunner(
+            new NullLogger()
+        );
+    }
```

Comment:
Use a copy of this [PipelineRunner](https://github.com/php-etl/csv-flow/blob/master/tests/functional/PipelineRunner.php)